### PR TITLE
Minor changes to trash.dm, add attack values to plate and tray

### PR DESCRIPTION
**Pull request by ghost**
- Add \improper to a few brand-name items (or at least their trash version) like Busta-Nuts or Syndicakes
- Update comments
- Trays now have a throwforce, a lower throw_range and throw_speed and a fairly decent force
- Plates now have a special attack proc to shatter one on people's head, dealing fair damage and destroying the plate. They can also be thrown for serious damage (equivalent to floor tiles)
- You can now attack with all trash items, this doesn't matter much for most of them as they have no attack values though
- Add some custom attack_verbs

### CODE REVIEW

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -84,8 +95,24 @@
...
+		var/mob/living/L = M
+		L.apply_damage(20, BRUTE, "head")
+	playsound(get_turf(src), "shatter", 70, 1)
+	if(ishuman(M))
+		var/mob/living/carbon/H = M
+		H.Stun(3)
+		H.emote("scream",,, 1)
+	for(var/mob/O in viewers(M, null))
+		shake_camera(O, 2, 3)
+	user.drop_item(src)
```
Comment (ComicIronic) [inline code/game/objects/items/trash.dm]: The arg for drop_item is not the item to drop.
It's the place to put it. I'm genuinely surprised this didn't make a runtime when you placed the plate in itself.

Comment (dylanstrategie) [inline code/game/objects/items/trash.dm]: Well it didn't

**Comment by Probe1** [comment]
>Plates now have a special attack proc to shatter one on people's head, dealing fair damage and destroying the plate. They can also be thrown for serious damage (equivalent to floor tiles)

for what purpose

**Comment by dylanstrategie** [comment]
Give the Chef a reason to remove the plates, spice up bar fights

**Comment by 9600bauds** [comment]
It really, really is just going to end up in people using the kitchen as gallery shooting. You also can't pick up the plates after people until they put it down.

There's no logic behind thrown plates doing as much damage as a laser, there's no gameplay reason for it, and the plate doesn't even shatter when thrown. Floortiles being extremely lethal when thrown is already a borderline bug, I give a loud "no" to plates doing the same.

**Comment by dylanstrategie** [comment]
I'll lower throwing damage to ten then. It's about average for a strong item

Also, I wanted plates shattering when thrown, but there's no procs on item side that easily deals with throw impact (there's Bump(), but it fires too often and I don't wanna risk it)

**Comment by 9600bauds** [comment]
Did you try copying the functionality from beer bottles?

**Comment by ComicIronic** [comment]
>there's no procs on item side that easily deals with throw impact 

There's throw_impact().

**Comment by dylanstrategie** [comment]
I'll check this more thoroughly later

**Comment by 9600bauds** [comment]
Also as for the tag, I think 10 throw damage is fine.

File: code/game/objects/items/trash.dm [hunk 2]
```diff
@@ -84,8 +95,24 @@
...
+		L.apply_damage(15, BRUTE, "head")
+	playsound(get_turf(src), "shatter", 70, 1)
+	if(ishuman(M))
+		var/mob/living/carbon/H = M
+		H.Stun(3)
+		H.emote("scream",,, 1)
+	for(var/mob/O in viewers(M, null))
+		shake_camera(O, 2, 3)
+	user.drop_item(get_turf(src))
+	returnToPool()
```
Comment (eswordthecat) [inline code/game/objects/items/trash.dm]: It's returnToPool(src)

File: code/game/objects/items/trash.dm [hunk 3]
```diff
@@ -84,8 +95,24 @@
...
+	user.visible_message("<span class='danger'>[user] shatters \the [src] on [M]'s head</span>", "<span class='warning'>You shatter \the [src] on [M]'s head</span>")
+	if(isliving(M))
+		var/mob/living/L = M
+		L.apply_damage(15, BRUTE, "head")
+	playsound(get_turf(src), "shatter", 70, 1)
+	if(ishuman(M))
+		var/mob/living/carbon/H = M
+		H.Stun(3)
+		H.emote("scream",,, 1)
+	for(var/mob/O in viewers(M, null))
```
Comment (eswordthecat) [inline code/game/objects/items/trash.dm]: Why mobs viewing the hitted mob needs their screen to be shake?

Comment (eswordthecat) [inline code/game/objects/items/trash.dm]: Also the syntax of viewers is number then object, not the other way around.

Comment (eswordthecat) [inline code/game/objects/items/trash.dm]: Also the observers will have their screen shaken!

**Comment by dylanstrategie** [comment]
Consider the item's balancing, should be fine now

**Comment by dylanstrategie** [comment]
Ay

**Pull request closed without merging**